cmd/test/cmd.go

```diff
@@ -380,9 +380,12 @@ func doRun(ctx context.Context, servicesToTest []string, options *Options, ioCtr
 
 func shouldRunInRemote(manifestDeploy *model.DeployInfo) bool {
 	if manifestDeploy != nil {
-		if manifestDeploy.Image != "" || manifestDeploy.Remote {
+		if manifestDeploy.Image != "" {
```

Comment:
We are not checking the value of the env var here, It doesn't seem correct but I might be missing something

Reply:
Ok, this works because this is changing the `--remote` flag for the deploy command, but we are not changing the value `RunInRemoteSet`, so it is not considering as the value to apply. Which seems odd.

I don't know if we should calculate here if the deploy has to be executed remotely or not. I'd leave it empty and rely on the deploy command to do the calculation, as we do for `okteto up`. If not, I would do the full calculation here setting also `RunInRemoteSet` to be consistent. As it is now, it is a bit misleading imo

Reply:
i am looking into this. The test cmd is requiring this information to set the deploy.Options (as if the flag was being used)

The logic that sets if is deploy remote or not is being used when GetDeployer, so i think this might not be needed as the calc to run remote is done there. I will check it and get back. thanks!

Reply:
after looking into the code, updated the PR using in test the same criteria as in deploy cmd @andreafalzetti please take a look as initial implementor to check the changes make sense. thanks!